pkgs/by-name/de/desktoppr/package.nix

```
@@ -0,0 +1,44 @@
+{
+  swiftPackages,
+  fetchFromGitHub,
+  swift,
+  swiftpm,
+  lib,
+}:
+
+swiftPackages.stdenv.mkDerivation (finalAttrs: {
```

```suggestion
stdenv.mkDerivation (finalAttrs: {
```

Why `swiftPackages.stdenv.mkDerivation`?

---

Please read:

https://github.com/NixOS/nixpkgs/blob/b0998a53ed83c1424d9fa396e556cc163eca2a41/doc/languages-frameworks/swift.section.md?plain=1#L50-L127